projects/app/boards/stm32f401re/STM32F401RETx_FLASH.ld

```diff
@@ -62,7 +62,7 @@ _Min_Stack_Size = 0x400; /* required amount of stack */
 MEMORY
 {
 RAM (xrw)      : ORIGIN = 0x20000000, LENGTH = 96K
-FLASH (rx)      : ORIGIN = 0x8000000, LENGTH = 512K
+FLASH (rx)      : ORIGIN = 0x8008000, LENGTH = 480K /* 512K */
```

Comment:
Define symbols for start address and length for both bootloader and applications, in order to make them readable from bootloader's source code. This change will make jumping to application dynamic, based on the value inside the linker file.

Reply:
The 0x8008000 information exists on the linker script of the actual application. From the bootloader side we dont have access to this and we cannot provide this information to the application.
1) Bootloader will eventually (try to) jump to a specific flash address (0x8008000).
2) Application will adjust its linker script to 0x8008000, to align with the bootloader requirement.
I dont see how defining the symbol on the bootloader linker script, or on app linker script will ease the integration.

Reply:
The whole point is to avoid the 0x8008000 completely, by just exposing the address as APP_CODE_BANK1_START_ADDRESS or smth. Of course this symbol will exist at both linker scripts. The user must adjust it at both linker script files if he wishes to.

Reply:
Let's leave it for a next PR